code/modules/reagents/newchem/pyro.dm

```diff
@@ -69,7 +69,7 @@
 	if(method == TOUCH && isliving(M))
 		M.adjust_fire_stacks(5)
 		M.IgniteMob()
-		new /obj/fire(M.loc)
```

Comment:
not burninator: the chem: the 3d experience: on ice: reloaded: remastered for blu-ray enough for me